test/core/client_channel/xds_override_host_lb_config_parser_test.cc

```diff
@@ -33,19 +33,7 @@ namespace grpc_core {
 namespace testing {
 namespace {
 
-class XdsOverrideHostConfigParsingTest : public ::testing::Test {
- public:
-  static void SetUpTestSuite() {
-    grpc_core::SetEnv("GRPC_EXPERIMENTAL_XDS_ENABLE_HOST_OVERRIDE", "TRUE");
-    grpc_init();
-  }
-
-  static void TearDownTestSuite() {
-    grpc_shutdown_blocking();
-    grpc_core::UnsetEnv("GRPC_EXPERIMENTAL_XDS_ENABLE_HOST_OVERRIDE");
-    grpc_core::CoreConfiguration::Reset();
-  }
-};
+class XdsOverrideHostConfigParsingTest : public ::testing::Test {};
```

Comment:
Since we no longer need to do anything in this test suite, we can remove it completely and change all of the `TEST_F()` invocations below to be `TEST()` instead.